nncf/common/graph/__init__.py

```diff
@@ -10,6 +10,3 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
-
-from nncf.common.graph.graph import NNCFNode
-from nncf.common.graph.graph import NNCFGraph
```

Comment:
What is the reason for these changes?

Reply:
This seems to have happened automatically during IDE refactoring, but the comment above in a reply to @ljaljushkin 
 stands - if we specify something in an `__init__.py` script, then it will be automatically imported during an import from any subpackage of `nncf.common.graph`. Do we really want to implicitly import `NNCFGraph` every time that we, for example, `from nncf.common.graph.layer_attributes import ConvolutionLayerAttributes`?

Reply:
Done